packages/sdk/src/modules/JsonRpcDatasource.ts

```diff
@@ -119,22 +129,17 @@ export default class JsonRpcDatasource extends BaseDatasource {
     next = null
   }: FetchUnspentUTXOsOptions) {
     if (!address) {
-      throw new Error("Invalid address")
+      throw new Error("Invalid request")
     }
 
     let utxos: UTXO[] = []
     do {
       const { unspents, pagination } = await rpc[this.network].call<{
         unspents: UTXO[]
-        pagination: {
-          limit: number
-          prev: string | null
-          next: string | null
-        }
+        pagination: JsonRpcPagination
```

Comment:
Does this now allow for configurable pagination?
Are there options for looping through pages to get all unspents or is it still limited to one hard-coded call of up-to 50 utxos? Think we might need to have some further discussions about the pagination of unspents ...

Reply:
This fn will now loop until there are UTXOs to be fetched wrt. to the limit (default: 50) and next page. After that, it will return all the UTXOs as response.